polars/polars-core/src/chunked_array/temporal/date.rs

```diff
@@ -30,8 +30,9 @@ impl DateChunked {
         Int32Chunked::from_vec(name, unit).into()
     }
 
-    /// Format Date with a `format` rule. See [chrono strftime/strptime](https://docs.rs/chrono/0.4.19/chrono/format/strftime/index.html).
-    pub fn strftime(&self, format: &str) -> Utf8Chunked {
+    /// Convert from Date to Utf8 with the given format.
+    /// See [chrono strftime/strptime](https://docs.rs/chrono/0.4.19/chrono/format/strftime/index.html).
+    pub fn to_string(&self, format: &str) -> Utf8Chunked {
```

Comment:
Why are these renamed? 

Shouldn't we just be able to add 2 aliases?

Reply:
We can! I had it like that initially, but aren't these always accessed through the namespace? In that case, we only need the alias on the namespace level (which is how it is now).

But if these methods on the chunked array are called directly by users then we should keep the `strftime` method.

Reply:
@ritchie46 Could you get back to me on this one? I will update the PR accordingly.